### ISSUE TITLE
Add a label to the rich text editor <textarea> element

### ISSUE THREAD

**Issue by tbolt** [CMSgov/eAPD#2546]
This is to address the rich text editor not announcing anything to the screen reader when it is tab-navigated to.

Rohit's findings/solution:
Check 95: textarea element missing an associated label.
Repair: Add a label element immediately before or after the textarea element. Set the for attribute value of the label element to the same value as the id attribute value of the textarea element. Add label text to the label element. Or, set the title attribute value to the textarea element to the label text. Or, add a label element that surrounds the textarea element and add label text.
Describe the task here and link or add dependencies to any applicable issues...

### This task is done when...
 - [ ] Rich text editing boxes (TinyMCE) components are updated to address the above concern

**Comment by radavis** [comment]
I filed an issue with the tinymce-react library. It isn't clear how we can label the tinymce editor as a form field that a screen reader will announce the label. https://github.com/tinymce/tinymce-react/issues/177

Typically, you simply add a `<label>` tag with a `for` attribute to point at the form field id. However, this doesn't achieve the desired results .

There are some other things/hacks I can try in the meantime.
